src/vm/ClrEtwAll.man

```diff
@@ -3573,6 +3610,11 @@
                            task="AssemblyLoader"
                            symbol="AssemblyLoadFromResolveHandlerInvoked" message="$(string.RuntimePublisher.AssemblyLoadFromResolveHandlerInvokedEventMessage)"/>
 
+                    <event value="296" version="0" level="win:Informational"  template="ResolutionAttempted"
```

Comment:
Can you make this 292 instead? Sorry, I had left a gap for this event since I thought that order made more sense, but then forgot to tell you.

Reply:
Sure.  I should have seen the gap.